.github/workflows/docs-ci.yml

```diff
@@ -0,0 +1,38 @@
+name: CI
+on:
+  push:
+    branches: [main]
+    paths:
+        - 'docs/**'
+
+  pull_request:
+    branches: [main]
+
```

Comment:
Shouldn't we also add the same `paths` for `pull_request`?

Reply:
Ooh good catch!